__tests__/app.test.js

```diff
@@ -0,0 +1,34 @@
+const request = require('supertest')
+const app = require('../app')
+const data = require('../db/data/test-data/index')
+const seed = require('../db/seeds/seed')
+
+beforeEach(() => {
+    return seed(data);
+})
+
+describe('GET Reviews', () => {
+    //console.log(reviewsData);
+    test('should return object with one entry: {"reviews" : <all reviews data>}', () => {
+        return request(app).get('/api/reviews')
+        .expect(200)
+        .then((res) => {
+            const reviews = res.body.reviews;
+
+            expect(Array.isArray(reviews)).toBe(true);
```

Comment:
No need to test to see if this is an array, you do it implicitly by checking the length and then looping through it